src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -293,34 +289,6 @@ final case class EServicesApiServiceImpl(
     }
   }
 
-  private def getProducerEServicesIds(producerId: UUID, consumersUUIDs: List[UUID])(implicit
-    contexts: Seq[(String, String)]
-  ): Future[List[UUID]] = getAllAgreements(producerId, consumersUUIDs)
-    .map(_.map(_.eserviceId).distinct)
-
-  private def getAllAgreements(producerId: UUID, consumerUUIDs: List[UUID])(implicit
```

Comment:
⚠️ this is now used by a recently merged PR

Reply:
Ok, I'm going to rebase.